### ISSUE TITLE
Is there anyway to install the latest mail-kit in Visual Studio 2010

### ISSUE THREAD

**Issue by truane** [jstedfast/MailKit#550]
When I try to install it i get -

The 'MailKit 1.18.0' package requires NuGet client version '2.12' or above, but the current NuGet version is '2.8.60318.667'.

but I can't upgrade to 2.12 without going to a later version of visual studio is there any way round it?

**Comment by jstedfast** [comment]
Sorry, thought I had already responded to this. I don't really understand why upgrading to 2.12 won't work with VS2010 :(

I thought the only difference was that newer nuget knew how to deal with the new .netstandard framework names whereas 2.8.x didn't. The API should have been the same?

Is it not possible to just download nuget 2.12's exe and copy it over the one that is currently in your system? That's how I upgrade nuget.

**Comment by truane** [comment]
No you can’t install 2.12 nugget in vs 2010 (or at least I couldn’t find one) I’m now looking at moving all my code to Visual Studio 2017 Enterprise but it’s no quick change

Thomas Ruane
Technical Consultant

**Comment by jstedfast** [comment]
There used to be a download link for the 2.12 nuget command line tool at https://www.nuget.org/downloads

I wasn't talking about the VSIX installer for Visual Studio.

You could try using the 3.x or 4.x nuget.exe from the download links.

**Comment by Flash1705** [comment]
I do not want to re-open this subject, I'm just adding a workaround which I found which might help somebody else.

I experienced the same problem with VS2010 and not wanting to change my project I found the following workaround.
I installed Community VS2017. Created a project targeting .NET4. I then used Nuget Manager in VS2017 to install MailKit and Mimekit.
I then copied the "packages" folder over to my existing VS 2010 Project.
I imported the 3 dll files: BouncyCastle.Crypto, MailKit & MimeKit, the last two files from  the "net40" folder as per my VS2017 project.
This worked perfectly for me, I hope this will help somebody.
